src/main/java/htsjdk/samtools/TextTagCodec.java

```diff
@@ -149,12 +150,12 @@ public String encodeUntypedTag(final String tagName, final Object value) {
      */
     public Map.Entry<String, Object> decode(final String tag) {
         final int numFields = StringUtil.splitConcatenateExcessTokens(tag, fields, ':');
-        if (numFields != TextTagCodec.NUM_TAG_FIELDS) {
+        if (numFields != TextTagCodec.NUM_TAG_FIELDS && numFields != TextTagCodec.NUM_TAG_FIELDS - 1) {
```

Comment:
Hi Yossi, that's not possible given the contract of StringUtil.splitConcatenateExcessTokens, but again, not a big deal.